test/nightly/deploy.test.ts

```diff
@@ -41,6 +41,11 @@ suite('Create Web App and deploy', async function (this: ISuiteCallbackContext):
         await testCreateWebAppAndDeploy(['Linux', 'Node 12 LTS'], testFolderPath);
     });
 
+    test('.NET Core LTS', async () => {
```

Comment:
Could you make a new file to deploy.dotnet and deploy.nodejs rather than having them all in one file?  This will get big very quickly 😅

Reply:
As an alternative, we could keep everything in this file and prevent it from getting too big with a strategy like [this](https://github.com/microsoft/vscode-azurefunctions/blob/master/test/nugetUtils.test.ts#L10). I expect `ITestCase` would look something like this:
```
interface ITestCase {
    workspaceFolder: string;
    runtimes: string[];
}
```

And the `for` loop would look something like this:
```
for (const testCase of testCases) {
    for (const runtime of testCase.runtimes) {
        test(runtime, async () => {
            const testFolderPath: string = await getWorkspacePath(testCase.workspaceFolder);
            await testCreateWebAppAndDeploy(['Linux', runtime], testFolderPath);
        });
    }
}
```